SNIPS/snip-75.md

```diff
@@ -0,0 +1,175 @@
+---
+snip: 75
+title: Starknet Security Council (SSC)
+author: Elias Tazartes <@Eikix>, Mattéo Georges <@mattegoat>, Patrícia Gil de Brolezzi <@pat_zip>, Luca Donno <@lucadonnoh>
```

Comment:
Need to add one last author: luiz-lvj <lvj253@gmail.com>
Waiting to get his full name